### PR TITLE
Change contact info text

### PR DESCRIPTION
**Pull request by murrayholmes96**
Issue: Scottish-Natural-Heritage/Licensing#657

### CODE REVIEW

**Comment by murrayholmes96** [comment]
Reverted the text that i changed by mistake

File: src/views/start.njk [hunk 1]
```diff
@@ -54,18 +54,40 @@
         If you need assistance with this application, contact the NatureScot Licensing Team.
```
Comment (JohnJAMoore) [inline src/views/start.njk:54]: katherine would like this text to be - 'If you need assistance with this service, contact the NatureScot Licensing Team'

**Review by JohnJAMoore — COMMENTED** [review]
Hey all looks good just a little text change requested by katherine dude

**Review by JohnJAMoore — APPROVED** [review]
LGTM 👍

**Pull request merged**